tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "files": [],
   "references": [
     {
-      "path": "./packages/basic"
-    }
+      "path": "./sam-app"
```

Comment:
Question - do we want to modify this project name i.e. `auth`?

Reply:
Good idea